src/layouting/data_limits.jl

```diff
@@ -144,6 +144,9 @@ function iterate_transformed(plot)
 end
 
 function iterate_transformed(points, model, space, trans_func)
+    if space != :data
+        (,)
```

Comment:
```suggestion
        tuple()
```
I always make this mistake...